README.md

```diff
@@ -7,7 +7,7 @@ Official PHP bindings to the Intercom API
 
 ## Installation
 
-Requires PHP 5.6.
+This library supports PHP 7.3, 7.2 and 7.1.
```

Comment:
3... 2... 1... 🚀

I would suggest a copy amend here, makes it easier to read and keeps it open for future cases:
```suggestion
This library supports PHP 7.1 and later
```